azurerm/internal/services/compute/windows_virtual_machine_resource.go

```diff
@@ -153,6 +153,11 @@ func resourceWindowsVirtualMachine() *schema.Resource {
 				Default:  true,
 			},
 
+			"encryption_at_host": {
```

Comment:
Also I suppose we could rename this to `encryption_at_host_enabled` given that this is a switch, what do you think?